loadouts/germany/loadouts/README.md

```diff
@@ -1 +1,33 @@
-# Loadouts
\ No newline at end of file
+# Loadouts
+
+## 2024 Jäger Squad, German Armed Forces
+
+**File:** `deu_2024_jäger_flecktarn_v5.sqf`
+
+**Modpack Version:** `[V5]`
+
+**Template Version:** `1.3`
+
+**Roles:** 
+- `RFL` Rifleman
+- `SL` Squad Leader
+- `FTL` Fireteam Leader
+- `MED` Medic
+- `AR` Autorifleman
+- `LAT` Anti-Tank, Light
+- `GRD` Grenadier
+- `RTO` Radio Telephone Operator
+
+**Description:**
+<!-- Description -->
+This loadout is depicting a modern German light infantry unit. They are called "Jäger" in German. Mostly using Boxer GTK AFV's for transportation and light support.
+
+**Country:** Germany
+
+
+**Associated Files:**
+None
+
+<a href="https://github.com/clustermod/HCMF3-Loadouts/blob/addloadouts/loadouts/germany/loadouts/deu_2024_jäger_flecktarn_v5.sqf">
```

Comment:
Keep in mind that when this PR is merged and it's branch deleted this link will no longer work. To fix this just replace `addloadouts` with `master`.

Reply:
Actually this link already doesn't work since the `addloadouts` branch doesn't exist

Reply:
fixed